e2e/cephfs.go

```diff
@@ -402,6 +406,80 @@ var _ = Describe(cephfsType, func() {
 				}
 			})
 
+			By("create a PVC and check PVC/PV metadata on CephFS subvolume", func() {
```

Comment:
Add e2e for pvc restore and pvc clone.

Reply:
This will be part of a different PR: https://github.com/ceph/ceph-csi/pull/3168